<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorPreferences</class>
 <widget class="QWidget" name="SimulatorPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>267</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="randomGroup">
     <property name="title">
      <string>&amp;Random run</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QLabel" name="stepsLabel">
        <property name="text">
         <string>&amp;Number of steps:</string>
        </property>
        <property name="buddy">
         <cstring>stepsSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="stepsSpin"/>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="randomBox">
        <property name="text">
         <string>R&amp;andom seed</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="seedLabel">
        <property name="text">
         <string>&amp;Seed:</string>
        </property>
        <property name="buddy">
         <cstring>seedSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="seedSpin">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="delayLabel">
        <property name="text">
         <string>&amp;Delay [ms]:</string>
        </property>
        <property name="buddy">
         <cstring>delaySpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="delaySpin">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colourGroup">
     <property name="title">
      <string>&amp;Transition colours</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="normalLabel">
        <property name="text">
         <string>N&amp;ormal:</string>
        </property>
        <property name="buddy">
         <cstring>normalButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="highlightLabel">
        <property name="text">
         <string>&amp;Highlighted:</string>
        </property>
        <property name="buddy">
         <cstring>highlightButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtColorPicker" name="highlightButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtColorPicker" name="normalButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QPushButton</extends>
   <header location="global">qtcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stepsSpin</tabstop>
  <tabstop>delaySpin</tabstop>
  <tabstop>seedSpin</tabstop>
  <tabstop>randomBox</tabstop>
  <tabstop>normalButton</tabstop>
  <tabstop>highlightButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
